addons/point_of_sale/static/tests/tours/ProductConfigurator.tour.js

```diff
@@ -43,7 +43,7 @@ registry.category("web_tour.tours").add("ProductConfiguratorTour", {
 
             // Check that the product has been added to the order with correct attributes and price
             ProductScreen.selectedOrderlineHas(
-                "Configurable Chair (Red, Metal, Other: Custom Fabric)",
```

Comment:
ideally we should still check that the correct attributes are shown.
Maybe you can add a new option in `OrderWidgetMethods.js`